### PR TITLE
Remove cancellationToken.js

### PR DESCRIPTION
**Pull request by jakebailey**
`cancellationToken.js` is a file which contains an implementation of a file-based cancellation token. It is loaded dynamically by `tsserver.js`, but nowhere else. As far as I can tell, there's no reason to not just include its code in `tsserver.js` directly and drop the extra file and `require`.

(Going through the repo history also doesn't indicate why this was a separate file; perhaps there's a good reason for this to exist and I lack context.)

### CODE REVIEW

**Review by sandersn — APPROVED** [review]
No reason to have it separate that I know of.

**Comment by jakebailey** [comment]
@typescript-bot perf test startup-only

File: src/tsserver/nodeServer.ts [hunk 1]
```diff
@@ -668,3 +661,60 @@ function startNodeSession(options: StartSessionOptions, logger: ts.server.Logger
...
+}
+
+function createCancellationToken(args: string[]): ts.server.ServerCancellationToken {
+    let cancellationPipeName: string | undefined;
+    for (let i = 0; i < args.length - 1; i++) {
+        if (args[i] === "--cancellationPipeName") {
+            cancellationPipeName = args[i + 1];
+            break;
+        }
+    }
```
Comment (mrazauskas) [inline src/tsserver/nodeServer.ts:683]: @jakebailey Perhaps `ts.server.findArgument()` can be used here as well?

https://github.com/microsoft/TypeScript/blob/30979c2651e5b0aa8ab583bbd8161cac3d0ea317/src/tsserver/nodeServer.ts#L94

Comment (jakebailey) [reply src/tsserver/nodeServer.ts:683]: https://github.com/microsoft/TypeScript/pull/60436

Comment (mrazauskas) [reply src/tsserver/nodeServer.ts:683]: That was quick! Thanks.

**Pull request merged**